src/fondue/Fondue.js

```diff
@@ -221,19 +226,29 @@ export default class Fondue {
 		return fvar;
 	}
 
+	// Return all color format tables.
+	// Usage:
+	//   fondue.colorFormats -> return e.g. ["SVG "] or empty array
+	get colorFormats() {
+		// const tables = Object.keys(this._raw()); // Why doesn't this work??
+		const tables = this._font.opentype.directory.map((d) => d.tag);
+		const colorTables = ["COLR", "sbix", "CBDT", "SVG "];
```

Comment:
This is also defined in `isColor` above. Can you refactor `isColor` to use this `colorFormats()`, or extract this list to prevent duplication?

Reply:
Good call! 👍